src/objects/job.py

```diff
@@ -96,9 +96,7 @@ def __init__(
             logs_dir=self.logs_dir,
             junit_dir=self.junit_dir,
         )
-        # self.has_test_failures = self._check_has_test_failures(failures=self.failures)
         self.has_test_failures = any(failure.failure_type == "test_failure" for failure in self.failures or [])
-        # self.has_pod_failures = self._check_has_pod_failures(failures=self.failures)
         self.has_pod_failures = any(failure.failure_type == "pod_failure" for failure in self.failures or [])
```

Comment:
why is `or []` needed?

Reply:
I guess `self.failures` can be None (which it shouldn't be, it must be set to [] in init)

also I think it's better to loop over `self.failures` and set the vars

````
for failure in self.failures:
    if failure.failure_type == "test_failure":
        self.has_test_failures = True
        break
    elif failure.failure_type == "pod_failure":
        self.has_pod_failures = True
        break
````

I know that `self.failures` will not going to be huge list but with the above code in some cases you break before the loop ends which is better in my opinion.

Reply:
@myakove I implemented your suggestion